Added a command TurnOnClimbMode

JalenEvans opened the pull request:
TurnOnClimbMode switches a boolean flag that allows the climb to worl.
Without it on, climb does not run. I also commented the intakeBtn and
uncommented deployBtn because intakeBtn was only menat for testing.

JalenEvans committed:
Added a command TurnOnClimbMode

TurnOnClimbMode switches a boolean flag that allows the climb to worl.
Without it on, climb does not run. I also commented the intakeBtn and
uncommented deployBtn because intakeBtn was only menat for testing.

hazeltorek commented on src/main/java/frc/robot/subsystems/ClimbSubsystem.java:
```diff
@@ -129,6 +131,13 @@ public void zeroRotatingArm()
+  /**
+   * Changes the current status of climbMode to th eopposite of what it was
+   */
+  public void setClimbMode() {
+    climbMode = !climbMode;
+  }
```
this is a toggle and not a set

j-hunter replied:
Also, be sure to still have a set and get method in addition to a toggle

JalenEvans committed:
Added a toggleClimMode and a getClimbMode

Changed the purpose of setClimMode to make it an actual setter and added
toggleCimbMode which toggles climbMode (Duh). Also added a getter for
climbMode.

j-hunter commented on src/main/java/frc/robot/commands/climb/TurnOnClimbMode.java:
```diff
@@ -0,0 +1,47 @@
+  @Override
+  public void end(boolean interrupted) {
+    climb.setClimbMode();
+    intake.retractIntake();
```
This needs to run in an execute loop to retract the intake

JalenEvans committed:
Changed the command to make it manually toggle

Really weird thing I did lol. I made it toggle climbMode inside of the
command and then either deploy or retract depending on the current state
of climbMode.

hazeltorek reviewed (approved):
fine i guess

hazeltorek commented on src/main/java/frc/robot/subsystems/ClimbSubsystem.java:
```diff
@@ -129,6 +131,28 @@ public void zeroRotatingArm()
+
+  /**
+   * Returns climbMode
+   * @return boolean
```
"true if climb is active, false otherwise"

JalenEvans committed:
Made better comments

hazeltorek commented:
merged as part of #45
